services/kubecost/0.34.0/defaults/cm.yaml

```diff
@@ -17,6 +17,7 @@ data:
           enabled: true
         grafana:
           enabled: true
+        priorityClassName: "dkp-high-priority"
```

Comment:
```suggestion
```

kubecost pulls in prometheus and grafana subcharts, we'll want to set priority classes for those deployments as well. For prometheus, [server](https://github.com/kubecost/cost-analyzer-helm-chart/blob/1723242ea6744b2d20cdda6dea9a4a59ffe22faa/cost-analyzer/charts/prometheus/values.yaml#L551) and [alertmanager](https://github.com/kubecost/cost-analyzer-helm-chart/blob/1723242ea6744b2d20cdda6dea9a4a59ffe22faa/cost-analyzer/charts/prometheus/values.yaml#L51). Doesn't look like they support setting priority classes for grafana yet, I'll look at adding it there too.